apps/studio/pages/project/[ref]/database/etl/index.tsx

```diff
@@ -20,35 +20,27 @@ const DatabaseReplicationPage: NextPageWithLayout = () => {
   }
 
   return (
-    <>
+    <PageLayout
+      size="large"
+      title="ETL Replication"
+      subtitle={
+        enablePgReplicate
+          ? 'Replicate your database to any analytics destination'
+          : 'Send data to other destinations'
```

Comment:
@imor why do we have two different subtitles depending on if `enablePgReplicate` is true?

Is this (new) first subtitle true for both?

> Replicate your database to any analytics destination

The second one is a bit vague in my opinion. Not a huge deal either way.